main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	spinhttp "github.com/spinframework/spin-go-sdk/http"
+
+	"github.com/fermyon/spin-redirect/redirect"
+)
+
+func init() {
+	r := redirect.NewSpinRedirect()
+	spinhttp.Handle(r.HandleFunc)
+}
+
+func main() {
```

Comment:
This should no longer be needed for the component.  Unless it's required for the tests or something?